### DISCUSSION TITLE
Define constants for fortran numeric types

### DISCUSSION THREAD

**Discussion by K20shores** [NCAR/musica#147] [Ideas]
Something like this, but for musica: https://github.com/NCAR/tuv-x/blob/23690d5944a8b54eba768cff9afd02e685bbfe02/src/util/constants.F90#L17-L20

**Comment by mattldawson** [comment]
After working with those constants in TUV-x for a while, I think the only ones that are actually useful are the float and double types (the others just lead to more typing without any real benefits that I've seen)

**Comment by mattldawson** [comment]
Also, should this be done in conjunction with exploring the possibility of passing fortran floating point numbers by reference to C functions without first copying to `c_double` variables?
